diff --git a/node_modules/@changesets/git/dist/git.cjs.dev.js b/node_modules/@changesets/git/dist/git.cjs.dev.js
index c7a8e91..b9e9d1a 100644
--- a/node_modules/@changesets/git/dist/git.cjs.dev.js
+++ b/node_modules/@changesets/git/dist/git.cjs.dev.js
@@ -88,7 +88,7 @@ async function getCommitsThatAddFiles(gitPaths, cwd) {
   do {
     // Fetch commit information for all paths we don't have yet
     const commitInfos = await Promise.all(remaining.map(async gitPath => {
-      const [commitSha, parentSha] = (await spawn__default['default']("git", ["log", "--diff-filter=A", "--max-count=1", "--pretty=format:%h:%p", gitPath], {
+      const [commitSha, parentSha] = (await spawn__default['default']("git", ["log", "--follow", "--diff-filter=A", "--max-count=1", "--pretty=format:%h:%p", gitPath], {
         cwd
       })).stdout.toString().split(":");
       return {
diff --git a/node_modules/@changesets/git/dist/git.cjs.prod.js b/node_modules/@changesets/git/dist/git.cjs.prod.js
index 7fc9a15..716117d 100644
--- a/node_modules/@changesets/git/dist/git.cjs.prod.js
+++ b/node_modules/@changesets/git/dist/git.cjs.prod.js
@@ -59,7 +59,7 @@ async function getCommitsThatAddFiles(gitPaths, cwd) {
   let remaining = gitPaths;
   for (;;) {
     const commitInfos = await Promise.all(remaining.map((async gitPath => {
-      const [commitSha, parentSha] = (await spawn__default.default("git", [ "log", "--diff-filter=A", "--max-count=1", "--pretty=format:%h:%p", gitPath ], {
+      const [commitSha, parentSha] = (await spawn__default.default("git", [ "log", "--follow", "--diff-filter=A", "--max-count=1", "--pretty=format:%h:%p", gitPath ], {
         cwd: cwd
       })).stdout.toString().split(":");
       return {
